service/storage_service.cc

```diff
@@ -471,7 +471,7 @@ class topology_mutation_builder {
     topology_mutation_builder& set(const char* cell, const std::unordered_set<dht::token>& value);
     topology_mutation_builder& set(const char* cell, const uint32_t& value);
     topology_mutation_builder& set(const char* cell, const utils::UUID& value);
-    topology_mutation_builder& set_transition_state(topology::transition_state);
+    topology_mutation_builder& set_transition_state(std::optional<topology::transition_state>);
```

Comment:
I think it will be nicer to have `del_transition_state()`

Reply:
ok